example_code/shmem_put_signal_example.c

```diff
@@ -0,0 +1,42 @@
+#include <stdio.h>
+#include <shmem.h>
+#include <stdlib.h>
+
+int main(void)
+{
+    int i, err_count  = 0;
+
+    shmem_init();
+
+    size_t    size    = (2<<10);
```

Comment:
`2<<10 == 1<<11 == 2048`.  Do you mean 1<<10, for 2^10?

Reply:
I would argue that writing `2 << 10` to get 2048 is poor programming practice and would rather see `1 << 11` or `2048`.

Reply:
fixed used 2048 directly